docs/site/downloads/cldr-45.md

```diff
@@ -1,3 +1,7 @@
+---
+title: CLDR 45 Release Note
+---
+
```

Comment:
@macchiati FYI this was missing in the PR #3650

Reply:
Thanks